docs/content/Configuration/Environment-Variables-Reference.md

```diff
@@ -125,31 +125,31 @@ databases [in this guide][link-connecting-to-db].
-| `CUBESTORE_S3_SUB_PATH`         | The path in a AWS S3 bucket to store pre-aggregations. Optional                                                                   | -                                                                               |
-| `CUBESTORE_SELECT_WORKERS`      | The number of Cube Store sub-processes that handle `SELECT` queries. Defaults to `4`                                              | A valid number                                                                  |
-| `CUBESTORE_SERVER_NAME`         | The full name and port number of the Cube Store server. Must be unique for each instance in cluster mode. Defaults to `localhost` | A valid address/port pair                                                       |
-| `CUBESTORE_WAL_SPLIT_THRESHOLD` | The maximum number of rows to keep in a single chunk of data right after insertion. Defaults to `262144`                          | A valid number                                                                  |
-| `CUBESTORE_WORKER_PORT`         | The port for Cube Store workers to listen to connections on. When set, the node will start as a **worker** in the cluster         | A valid port number                                                             |
-| `CUBESTORE_WORKERS`             | A comma-separated list of address/port pairs; for example `worker-1:3123,localhost:3124,123.124.125.128:3123`                     | A comma-separated list of address/port pairs                                    |
-| `SERVICE_ACCOUNT_JSON`          | A JSON string containing credentials for Google Cloud. Required when using Google Cloud Storage                                   | [The contents of a JSON credentials file for Google Cloud][link-gcp-creds-json] |
+| Environment variable            | Description                                                                                                                                          | Possible Values                                                                 |
+| ------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------- |
+| `CUBESTORE_BIND_ADDR`           | The address/port pair for Cube Store's MySQL-compatible interface. Defaults to `0.0.0.0:3306`                                                        | A valid address/port pair                                                       |
+| `CUBESTORE_DATA_DIR`            | A path on the local filesystem to store a local replica of the data. Defaults to `.cubestore/data`                                                   | A valid path on the local filesystem with read/write access                     |
```

Comment:
"...a local replica of the pre-aggregations data" maybe? Just trying to be more specific but not sure if it's needed here.